easybuild/easyconfigs/r/RStudio-Server/RStudio-Server-1.3.1093_fix-data-dir.patch

```diff
@@ -0,0 +1,18 @@
+commit https://github.com/ComputeCanada/easybuild-easyconfigs/commit/77ecdad8fc629078d106fba250951c5c8c22ae52
```

Comment:
I updated the patch, please see : https://github.com/ComputeCanada/easybuild-easyconfigs/commit/030686bba2c8c6acfb94d20b9988b21a4b23177e

Reply:
Updated in https://github.com/easybuilders/easybuild-easyconfigs/pull/12544/commits/a335b99d2dec1fbbf050cb24d2fd3300399640b7